src/pint/plot_utils.py

```diff
@@ -176,3 +177,127 @@ def phaseogram_binned(
         plt.close()
     else:
         plt.show()
+
+
+def plot_priors(
+    model,
+    chains,
+    fitvals,
+    fiterrs,
+    maxpost_fitvals,
+    burnin=100,
+    bins=100,
+    scale=False,
+    file=False,
+):
+
+    """
+    Show binned samples, prior probability distribution and an initial gaussian
+    probability distribution plotted with 2 sigma, maximum posterior and original
+    fit values marked.
+    
+    Parameters
+    ----------
+    model
+        Input timing model 
+    chains: dict    
+        Post MCMC integration chains that are input into a histogram and compared with the priors. Contain the fitkeys and the samples
+    fitvals
+        Initial Parameter values
+    fiterrs
+        Initial Parameter uncertainties
+    maxpost_fitvals
+        Maximum Posterior values
+    With scale is True
+        The priors and initial probability distributions are scaled to the peak of the histogram.
+    With scale is False
+        The priors and initial probability distributions are plotted without the scaling.
+    """
+    keys = []
+    values = []
+    for item in chains.items():
```

Comment:
With `.items()` it's more conventional to write `for k, v in chains.items():`

Reply:
I will update that